open_flamingo/train/train.py

```diff
@@ -218,6 +233,8 @@ def main():
 
     args = parser.parse_args()
 
+    args.local_rank = int(os.environ.get("LOCAL_RANK", -1))  # for deepspeed
```

Comment:
Hmm, just to understand what's happening here -- deepspeed populates this environment variable, which we load, but then our code overwrites this variable based on Slurm logic on line 272.  Are we sure these match?